godot-macros/src/class/data_models/field_export.rs

```diff
@@ -35,9 +35,11 @@ pub enum FieldExport {
         or_greater: bool,
         or_less: bool,
         exp: bool,
+        radians_as_degrees: bool,
         radians: bool,
         degrees: bool,
         hide_slider: bool,
+        suffix: Option<String>,
```

Comment:
Can't you just store the `TokenStream` and convert it later? Kinda weird that `min`, `max`, and `step` are all expression but `suffix` isn't.

Reply:
Yes, that seems to save one `to_string` call and solves a question I had